### ISSUE TITLE
WTF do these strings even mean?

### ISSUE THREAD

**Issue by mbabker** [joomla/joomla-cms#14557]
```ini
COM_JOOMLAUPDATE_UPDATE_LOG_CONFIRM_FINALISE="The confirm for the finalise step has successfully passed."
COM_JOOMLAUPDATE_UPDATE_LOG_CONFIRM_FINALISE_FAIL="The confirm of the finalise step has failed"
```

1) These are writing to a log file.  Why are we translating log messages?
2) Anyone who can write these in a form of English that I might be able to read wins a beer.  I'll even take Spanish at this point, my broken understanding of it is probably enough to work with.

**Comment by zero-24** [comment]
It come from me. Feel free to replace it with a different message. It should just show in the logs that we bypassed the token checks here.

**Comment by mbabker** [comment]
Since I'm mildly annoyed at the moment (sorry), it's not my intent to trash anyone's English.  Just so it's clear.  I'm just mildly annoyed chasing down ghost strings and trying to decipher messages.

I'm only doing the en-US variant on Crowdin right now and I'm struggling in places.  And I'm supposed to be able to speak English (so the US school system says, others may feel differently).  This isn't helping our translators.

**Comment by zero-24** [comment]
I agree. So we can fix this by a native speaker suggests a better string and we use this. ;)

Btw it was used as language string as the other log messages in com_joomlaupdate got translated too e.g.
https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_joomlaupdate/models/default.php#L315

We can also completly remove that string / translation.

**Comment by brianteeman** [comment]
Closed as we have a PR #14564  I hope I understood it correctly
